ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx

```diff
@@ -67,11 +69,28 @@ function stringHashCode(str: string) {
     return hash;
 }
 
+function getPodBackgroundColor(podName: string) {
+    return POD_BACKGROUND_COLORS[Math.abs(stringHashCode(podName) % POD_BACKGROUND_COLORS.length)];
+}
+
 // ansi color for pod name
 function podColor(podName: string) {
     return colors[Math.abs(stringHashCode(podName) % colors.length)];
 }
 
+const PodLegend: React.FC<{logs: LogEntry[]}> = ({logs}) => {
```

Comment:
this would be on top of the log-viewer so in case if there are lot of pods, can have like a Horizontal scroll?